include/oneapi/mkl/detail/backends_table.hpp

```diff
@@ -58,6 +58,7 @@ static std::map<domain, std::map<device, std::vector<const char*>>> libraries =
           } },
         { device::nvidiagpu,
           {
+/// TODO: ENABLE_CUDA_BACKEND?
```

Comment:
I think it's better to keep this fine granularity since CUDA libraries are separated by domains, unfortunately we can't do it for  oneMKL, since it has all domains in one library.

Reply:
Do we need to remove TODO comments before merge?

Reply:
Agreed. This comment is a relic.